test/api/unit/services/SQS.test.js

```diff
@@ -101,6 +101,18 @@ describe("SQS", () => {
         }).catch(done)
       })
 
+      it("should respect the path component of a custom domain when setting BASEURL in the message", done => {
```

Comment:
I amended the commit. This test and the test without a path component both have an array of domains they iterate over to make sure that all return the same correct result.